Cython/Includes/cpython/datetime.pxd

```diff
@@ -6,22 +6,73 @@ cdef extern from "Python.h":
 
 cdef extern from "datetime.h":
     """
-    #if PY_MAJOR_VERSION < 3 && !defined(PyDateTime_DELTA_GET_DAYS)
-    #define PyDateTime_DELTA_GET_DAYS(o)         (((PyDateTime_Delta*)o)->days)
-    #endif
-    #if PY_MAJOR_VERSION < 3 && !defined(PyDateTime_DELTA_GET_SECONDS)
-    #define PyDateTime_DELTA_GET_SECONDS(o)      (((PyDateTime_Delta*)o)->seconds)
-    #endif
-    #if PY_MAJOR_VERSION < 3 && !defined(PyDateTime_DELTA_GET_MICROSECONDS)
-    #define PyDateTime_DELTA_GET_MICROSECONDS(o)   (((PyDateTime_Delta*)o)->microseconds)
+    #if PY_MAJOR_VERSION < 3
+        #if PY_MINOR_VERSION < 3
```

Comment:
Note that the original `#if` was meant for Python 2.x, that's why it says `PY_MAJOR_VERSION < 3`. You can use `PY_VERSION_HEX` to detect concrete Python versions.

Reply:
I solved this by just skipping all of that, they're now all `#ifndef`.